packages/libs/react-ui/src/components/Select/Select.tsx

```diff
@@ -34,16 +38,16 @@ export const Select: FC<ISelectProps> = forwardRef<
     disabled = false,
     children,
     ariaLabel,
+    variant = 'form',
     ...rest
   },
   ref,
 ) {
   return (
     <div
-      className={classNames(
-        containerClass,
-        disabled ? containerClassDisabled : '',
-      )}
+      className={classNames(selectVariants[variant], containerClass, {
```

Comment:
nit: I would order the classNames in order of specificity. So since containerClass is the most generic, it should go first so that it can be overridden (I think) by the following classnames. This might be why you needed to remove the backgroundColor from containerClass

Just a note for the future - might help with styling!